components/director/internal/domain/apptemplate/service.go

```diff
@@ -243,3 +350,91 @@ func (s *service) PrepareApplicationCreateInputJSON(appTemplate *model.Applicati
 	}
 	return appCreateInputJSON, nil
 }
+
+func (s *service) exists(ctx context.Context, inputName string, inputRegion interface{}) (bool, error) {
```

Comment:
The implementation of this function is identical to the GetByNameAndRegion function?

Reply:
yep. I had a note about that, but somehow i've missed it.